Jenkinsfile

```diff
@@ -247,6 +248,10 @@ node ('master') {
           if (x64linux_compiler_list.isEmpty() && coverage ){
             coverage_mac = true
           }
+          // TODO 2019-08-14 lebdron: IR-600 Fix integration tests execution when built with shared libraries
+          if (!x64linux_compiler_list.isEmpty() && build_shared_libs) {
+            testList = '()'
+          }
```

Comment:
This block can be deleted, If one decide to run with `custom_cmd` it is his responsibility to set the correct  `x64linux_compiler_list` and `testList`, we only mark the conflicting parameters with red color. we have number of parameters that can conflict. See `.jenkinsci/text-variables.groovy`

Reply:
There can be a case when developers want to test that build fails if you set different combinations of `testList` and `build_shared_libs` and currently here it is not possible.